src/tabs-motion/styled-components.js

```diff
@@ -79,13 +79,15 @@ export const StyledTab = styled<{
   $orientation?: OrientationT,
   $fill?: FillT,
   $focusVisible?: boolean,
+  $isActive?: Boolean,
```

Comment:
```suggestion
  $isActive?: boolean,
```

Reply:
Sorry about that, must have been autocomplete. Will fix.